Packs/RecordedFutureASI/Integrations/RecordedFutureASI/RecordedFutureASI.py

```diff
@@ -186,6 +533,10 @@ def fetch_incidents(client: Client, last_run: Dict[str, int]) -> Tuple[Dict[str,
 def main() -> None:
     api_key = demisto.params().get('apikey')
     project_id = demisto.params().get('project_id')
+    is_by_host = demisto.params().get('issue_grouping') == 'By Host'
+    expand_issues = demisto.params().get('expand_issues') == 'True'
+    incident_limit = int(demisto.params().get('max_fetch', DEFAULT_HOST_LIMIT))
+    min_severity = demisto.params().get('min_severity', DEFAULT_MIN_SEVERITY)
```

Comment:
Since demisto function calls are expensive, we should want to call them as little as possible.

```suggestion
    params = demisto.params()
    api_key = params.get('apikey')
    project_id = params.get('project_id')
    is_by_host = params.get('issue_grouping') == 'By Host'
    expand_issues = params.get('expand_issues') == 'True'
    incident_limit = int(params.get('max_fetch', DEFAULT_HOST_LIMIT))
    min_severity = params.get('min_severity', DEFAULT_MIN_SEVERITY)
```